lib/jsdom/living/nodes/RadioNodeList-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const NodeListImpl = require("./NodeList-impl").implementation;
+
+class RadioNodeListImpl extends NodeListImpl {
+  get value() {
+    this._update();
+    const element = this._list.find(e => e._localName === "input" && e.type === "radio" && e.checked);
+    if (element && element.value !== "") {
+      return element.value ? element.value : "on";
+    }
+    return "";
+  }
+  set value(value) {
+    this._update();
+    const element = this._list.find(
+      // https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#dom-radionodelist-value "On setting..."
+      e => e._localName === "input" && e.type === "radio" && (e.value ? value === e.value : value === "on")
+    );
+    if (element) {
+      element.checked = true;
```

Comment:
As above, change the `_checkedness`, not `checked`.

Reply:
Hmm. Actually I suspect the spec is wrong here. Please leave as-is and add a comment linking to https://github.com/whatwg/html/issues/7612 .

If you change this to `_checkedness`, do any tests fail? If none do, then adding one would be super appreciated (but again not required). It would help us straighten out the spec situation.

Reply:
Yes, if you use `_checkedness` here then a couple tests in `radionodelist.html` will fail as multiple buttons will have their _checkedness set to true